### PR TITLE
Fix contentSignatures to apply with "metafeeds" prefix

### PR DESCRIPTION
**Pull request by staltz**
1st commit updates tests to show that the current implementation is failing, 2nd commit will fix to pass the tests.

### CODE REVIEW

**Comment by staltz** [comment]
Parking this PR for now because I'd like to find a way of decoupling ssb-bendy-butt from metafeeds, and that requires not hard-coding any such prefixes in the library.

**Comment by staltz** [comment]
From Matrix: "okay, I think the way of generalizing this is to change the prefix to `bendybutt`".

**Comment by cryptix** [comment]
I pushed 55d631f onto this but the key derivation needs to be fixed to use the seeds because the full keys are no longer included in the vectors.

**Comment by staltz** [comment]
That's good, I can derive the full keys manually and include it. Thx

**Comment by staltz** [comment]
@cryptix I'm having some difficulty reproducing the subfeed keys, but the metafeed key is built correctly. That's the strange thing, because the same algorithm `deriveFeedKeyFromSeed` (in `ssb-meta-feed`) is used for both cases.

Meta feed:

```js
deriveFeedKeyFromSeed(
  Buffer.from('7365633073656330736563307365633073656330736563307365633073656330','hex'), 
  'testfeed'
  'bendy butt',
).id
// '@b99R2e7lj8h7NFqGhOu6lCGy8gLxWV+J4ORd1X7rP3c=.bbfeed-v1'
```

Subfeed 1 (expected `@2ykZ7Pu69mL+UdM3dc6ASn5DFE6UIUM2zX+pRnvKo1c=.ed25519`):

```js
deriveFeedKeyFromSeed(
  Buffer.from('7365633073656330736563307365633073656330736563307365633073656330','hex'), 
  Buffer.from('2323232323232323232323232323232323232323232323232323232323232323','hex').toString("base64")
).id
// '@Oo6OYCGsjLP3n+cep4FiHJJZGHyqKWztnhDk7vJhi3A=.ed25519'
```

Subfeed 2 (expected `@/BH8wn3ttfUxuwPWB8rdCUclk3r76Fg6lKkL2fPUItI=.ggfeed-v1`)

```js
deriveFeedKeyFromSeed(
  Buffer.from('7365633073656330736563307365633073656330736563307365633073656330','hex'),
  Buffer.from('4242424242424242424242424242424242424242424242424242424242424242','hex').toString("base64")
).id
// '@FY5OG311W4j/KPh8H9B2MZt4WSziy/p+ABkKERJdujQ=.ed25519'
```

**Comment by cryptix** [comment]
My bad. The label prefix `"ssb-meta-feed-seed-v1:"` was applied twice.. Updated vectors incoming.

**Comment by cryptix** [comment]
Please try again with b05d2be <3

**Comment by staltz** [comment]
Tests restored! Thanks for the fixes. :)

**Comment by cryptix** [comment]
Cool! Glad they are on par now.

I'd prefer if and suggest that the code derives the keys, not writing them back to the file manually. Otherwise we will need to do this dance every time there is an update.

**Comment by staltz** [comment]
Ah right, I'll have to bring in `ssb-meta-feeds` as a devDep then. It might create circular dependency, or not.

**Comment by cryptix** [comment]
Nice! Purrfect!

File: test/vector.js [hunk 1]
```diff
@@ -1,42 +1,74 @@
...
-    }
-    const encoded = bb.encode(msgExtracted)
-
-    const jsonEncodeDecode = JSON.stringify(bb.decode(encoded), null, 2)
-    const decode = JSON.stringify(
-      bb.decode(Buffer.from(msg.EncodedData, 'hex')),
-      null,
-      2
+  const getHex = (obj) => obj.HexString
+  const [mfHex, sf1Hex, sf2Hex] = vec.Metadata.filter(getHex).map(getHex)
```
Comment (arj03) [inline test/vector.js:33]: Nice

**Review by arj03 — APPROVED** [review]

**Comment by staltz** [comment]
Let's go!

**Pull request merged**